### PR TITLE
[Compute] AZ and VMSS rolling upgrade update and tests

### PR DESCRIPTION
**Pull request by hyonholee**
https://github.com/Azure/azure-rest-api-specs/pull/1727

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Compute/Compute.Tests/Resources/DiskEncryptionPreRequisites.ps1 [hunk 1]
```diff
@@ -0,0 +1,185 @@
+﻿Param(
```
Comment (markcowl) [inline src/SDKs/Compute/Compute.Tests/Resources/DiskEncryptionPreRequisites.ps1]: Why is this in the repo?

**Review by markcowl — CHANGES_REQUESTED** [review]
Also, can you update the generate.cmd and include the metadata file that is generated when you runit?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Provide a clear description that describes the reason for the PR. Explain the change in detail.
Failing to do this, will result in PR not being reviewed.
This is a public repo and every PR should be clearly explained as to what the PR is all about.

**Comment by shahabhijeet** [comment]
@hyonholee pleaes reopen the same PR once you have addressed all feedback/questions.
Having a PR open for more than a week affects our SLA data.

**Pull request closed without merging**